src/coreclr/jit/lowerxarch.cpp

```diff
@@ -5109,6 +5299,54 @@ GenTree* Lowering::TryLowerXorOpToGetMaskUpToLowestSetBit(GenTreeOp* xorNode)
     return blsmskNode;
 }
 
+//----------------------------------------------------------------------------------------------
+// Lowering::TryCanonizeEmbBroadcastCandicate:
+//    Tries to canonize the operands of an embedded broadcast op.
+//
+// Arguments:
+//    node - `Create` node being lowered
+//
+// Notes:
+//    This function will canonize the input operands of a commutive and embedded broadcast op.
+//    The operand to be broadcasted will be put to the 2nd operand.
+//
+void Lowering::TryCanonizeEmbBroadcastCandicate(GenTreeHWIntrinsic* node)
```

Comment:
This seems like maybe something we should be handling as part of `IsContainableHWIntrinsicOp` so the existing swapping code will work as expected.